### ISSUE TITLE
NEH logo naming 

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#84]
While using VoiceOver utility, I noticed that the logo was being read as "neh logo" instead of the full name of the logo like it does for George Mason University. It should be named "National Endowment for the Humanities logo" instead so the user understands what is being tabbed over.
